### ISSUE TITLE
When no tasty tests called, `test-options` passes to `QuickCheck` instead of `tasty`

### ISSUE THREAD

**Issue by Renegatto** [Liqwid-Labs/plutus-extra#162]
If no tasty tests called, `OPTS` from `cabal test --test-options=OPTS` somehow gets to quickcheck, not to tasty, which seems like a bug.

**Comment by kozross** [comment]
I think this is the default behaviour for `tasty`, but it's certainly weird and needs investigating.

**Comment by gnumonik** [comment]
@Renegatto Could you provide a simple example of the  behavior you're experiencing (i.e. the OPTs you're passing and the output you're seeing)? I believe that would help me track down the cause of this.

**Comment by Renegatto** [comment]
@gnumonik no, at the moment I don't have any project to trigger this issue on.
Just pass this way any `tasty` flag that `QickCheck` wouldn't expect, and you'll get `QickCheck` error.

**Comment by kozross** [comment]
@gnumonik I think the best solution is to 'swallow' those flags, assuming that we can do this. If not, I actually feel this might be a `tasty` bug; if that's the case, there's not a lot we can do about it other than report.

**Comment by gnumonik** [comment]
@Renegatto After digging through the tasty source, it seems very likely that the issue is on their end. I'm reasonably sure that Koz is right and that this is the default behavior, and that there is not an easy way to override it. I believe that it has to do with the way the Tasty typeclass instances are defined in the tasty quickcheck module, I can elaborate more if you'd like. 

I could pursue this more if you wish; there may be a less-obvious or not-so-easy way to alter Tasty's behavior. Let me know if you think that would be worthwhile. If not, I'll file an issue report upstream.

**Comment by Renegatto** [comment]
I'm very happy if it will be just reported to `tasty`, it looks like the cleanest solution.
You're free to close this issue as soon, as you think you're done.
